crypto/cms/cms_lib.c

```diff
@@ -620,59 +620,91 @@ int CMS_add1_crl(CMS_ContentInfo *cms, X509_CRL *crl)
 STACK_OF(X509) *CMS_get1_certs(CMS_ContentInfo *cms)
 {
     STACK_OF(X509) *certs = NULL;
+
+    if (!ossl_cms_get1_certs_ex(cms, &certs))
+        return NULL;
+    if (sk_X509_num(certs) == 0) {
+        OSSL_STACK_OF_X509_free(certs);
```

Comment:
Just `sk_X509_free`? We don't need to "pop" anything since we know then number of entries is 0.